ui/operator_ui/main.py

```diff
@@ -321,7 +321,7 @@ def index():
     'users_visible': True,
     'databases_visible': True,
     'resources_visible': RESOURCES_VISIBLE,
-    'postgresql_versions': ['11','12','13','14'],
+    'postgresql_versions': ['11', '12', '13', '14'],
```

Comment:
why there is no 15 here?  (not sure what this file is about :) )

Reply:
this is for the UI where we should make 15 available only when it#s ready to be used